docs/src/component/requirements.md

```diff
@@ -153,10 +153,9 @@ The below is a list of requirements for the crate, but it does not prevent addit
    contains custom errors.
 5. `hob` module: This module may optionally exist if a new guided HOB type has been created for this component. The
    HOB module and associated guided HOB(s) should be made public such that it can be consumed by others if the need
-   arises. Any common or spec defined HOBs should be added to the associated crates (such as `patina_pi`,
```

Comment:
In the course of this update, I realized a few `patina_sdk` places need to be updated in the docs. That is tracked in https://github.com/OpenDevicePartnership/patina/issues/905.